.changeset/fifty-pots-greet.md

```diff
@@ -0,0 +1,5 @@
+---
+"astro": patch
+---
+
+The i18n redirects should consider the `build.format` configuration
```

Comment:
```suggestion
Makes i18n redirects take the `build.format` configuration into account
```

Reply:
Thanks for breaking the `tense`ion Florian! 😆